vsts/build.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
   - script: 'python -m pip install --upgrade pip'
     displayName: 'Update pip'
 
-  - script: 'python dev_setup.py'
+  - script: 'python env_setup.py --no_dev'
```

Comment:
I'm curious why you needed to add this.  Was pre-commit causing problems?

Reply:
Yup. Not entirely sure why, but also didn't have much problem coding around it - VSTS has no reason to install pre-commit anyway. So now there's a "--no_dev" mode that's just for running and testing.

---
In reply to: [251161237](https://github.com/Azure/azure-iot-sdk-python-preview/pull/43#discussion_r251161237) [](ancestors = 251161237)